test/fake_package/lib/example.dart

```diff
@@ -11,6 +11,7 @@ get y => 2;
 
 const String COLOR = 'red';
 const String COMPLEX_COLOR = 'red' + '-' + 'green' + '-' + 'blue';
+const ConstantCat MY_CAT = const ConstantCat('tabby');
```

Comment:
Please also add a case for `const String COLOR='green'` and other cases of random spacing.